src/main/kotlin/com/exactpro/th2/httpserver/server/options/ServerOptions.kt

```diff
@@ -41,27 +41,13 @@ interface ServerOptions {
 
     /**
      * @return executor service to use to run client-serving [Runnable]s. Each [Runnable] runs until
-     * the connection with the client is closed or lost.
+     * the connection with the client is closed or lost
      */
-    fun createExecutorService(): ExecutorService {
-        val threadCount = AtomicInteger(1)
-        return Executors.newFixedThreadPool(25) { runnable: Runnable? ->
-            val t = Thread(runnable)
-            t.isDaemon = true
-            t.name = "th2-http-server-" + threadCount.incrementAndGet()
-            t
-        }
-    }
-
-    fun prepareResponse(request: RawHttpRequest, response: RawHttpResponse<*>): RawHttpResponse<*> {
-        return response
-    }
+    fun createExecutorService(): ExecutorService
 
-    fun onRequest(request: RawHttpRequest) {
+    fun prepareResponse(request: RawHttpRequest, response: RawHttpResponse<*>): RawHttpResponse<*> = response
 
-    }
-
-    fun onResponse(request: RawHttpRequest, response: RawHttpResponse<*>) {
+    fun onRequest(request: RawHttpRequest) = Unit
 
-    }
+    fun onResponse(request: RawHttpRequest, response: RawHttpResponse<*>) = Unit
```

Comment:
We probably should store messages into MQ in these methods instead of `IResponseManager` implementations. This will ensure that requests and responses are stored regardless of loaded `IResponseManager` implementation